packages/elements/src/utils/getElementOffsets.js

```diff
@@ -42,17 +42,17 @@ import { getMediaSizePositionProps } from '@googleforcreators/media';
 
 const PRECISION = 1;
 
-export function getMediaBoundOffsets({ element }) {
+export function getElementOffsets(element) {
```

Comment:
So, I know that this function only makes sense for media elements (that have scale and focal values), but I've made it general anyway (just returning all zeros for non-media).